packages/docs/docs/exporting-from-figma.md

```diff
@@ -0,0 +1,71 @@
+---
+id: exporting-from-figma
+title: Exporting designs from Figma to Remotion
+---
+
+Say you're collaborating as a Frontend Engineer with a design team, and you want a way to get a real-time representation of some of the design assets you have on Figma.
```

Comment:
The main usecase is not necessarily Figma -> Web handoff I would say, it is that you design something static using Figma, then animate it. It does not have to be different people.